EngineBay.Auditing/AuditEntry/CreateAuditEntry.cs

```diff
@@ -1,34 +1,31 @@
-﻿
-namespace EngineBay.Auditing
+﻿namespace EngineBay.Auditing
 {
-
     using EngineBay.Core;
     using FluentValidation;
-    using System.Security.Claims;
 
-    public class CreateAuditEntry : ICommandHandler<CreateAuditEntryRequest, AuditEntryDto>
+    public class CreateAuditEntry : IClaimlessCommandHandler<CreateAuditEntryRequest, AuditEntryDto>
     {
-
         private readonly AuditingWriteDbContext dbContext;
         private readonly IValidator<CreateAuditEntryRequest> validator;
 
-        public CreateAuditEntry(AuditingWriteDbContext dbContext)
+        public CreateAuditEntry(AuditingWriteDbContext dbContext, IValidator<CreateAuditEntryRequest> validator)
         {
             this.dbContext = dbContext;
+            this.validator = validator;
         }
 
-        public async Task<AuditEntryDto> Handle(CreateAuditEntryRequest createAuditEntryRequest, ClaimsPrincipal user, CancellationToken cancellation)
+        public async Task<AuditEntryDto> Handle(CreateAuditEntryRequest inputParameters, CancellationToken cancellation)
```

Comment:
rename to createAuditEntryRequest or command.
I normally call this "command" since it flows nicely with commadnHandler 🤷